python/cugraph/structure/graph.py

```diff
@@ -128,203 +51,39 @@ def __init__(
-                self._replicate_transposed_adjlist()
-
-    def _replicate_edgelist(self):
-        client = mg_utils.get_client()
-        comms = Comms.get_comms()
-
-        # FIXME: There  might be a better way to control it
-        if client is None:
-            return
-        work_futures = replication.replicate_cudf_dataframe(
-            self.edgelist.edgelist_df, client=client, comms=comms
-        )
-
-        self.batch_edgelists = work_futures
-
-    def _replicate_adjlist(self):
-        client = mg_utils.get_client()
-        comms = Comms.get_comms()
-
-        # FIXME: There  might be a better way to control it
-        if client is None:
-            return
-
-        weights = None
-        offsets_futures = replication.replicate_cudf_series(
-            self.adjlist.offsets, client=client, comms=comms
-        )
-        indices_futures = replication.replicate_cudf_series(
-            self.adjlist.indices, client=client, comms=comms
-        )
-
-        if self.adjlist.weights is not None:
-            weights = replication.replicate_cudf_series(self.adjlist.weights)
-        else:
-            weights = {worker: None for worker in offsets_futures}
-
-        merged_futures = {
-            worker: [
-                offsets_futures[worker],
-                indices_futures[worker],
-                weights[worker],
-            ]
-            for worker in offsets_futures
-        }
-        self.batch_adjlists = merged_futures
-
-    # FIXME: Not implemented yet
-    def _replicate_transposed_adjlist(self):
-        self.batch_transposed_adjlists = True
-
-    def clear(self):
-        """
-        Empty this graph. This function is added for NetworkX compatibility.
-        """
-        self.edgelist = None
-        self.adjlist = None
-        self.transposedadjlist = None
-
-        self.batch_edgelists = None
-        self.batch_adjlists = None
-        self.batch_transposed_adjlists = None
-
-    def add_nodes_from(self, nodes, bipartite=None, multipartite=None):
-        """
-        Add nodes information to the Graph.
-
-        Parameters
-        ----------
-        nodes : list or cudf.Series
-            The nodes of the graph to be stored. If bipartite and multipartite
-            arguments are not passed, the nodes are considered to be a list of
-            all the nodes present in the Graph.
-        bipartite : str
-            Sets the Graph as bipartite. The nodes are stored as a set of nodes
-            of the partition named as bipartite argument.
-        multipartite : str
-            Sets the Graph as multipartite. The nodes are stored as a set of
-            nodes of the partition named as multipartite argument.
-        """
-        if bipartite is None and multipartite is None:
-            self._nodes["all_nodes"] = cudf.Series(nodes)
+    def __getattr__(self, name):
+        if self.Impl is None:
+            raise Exception("Graph is Empty")
```

Comment:
attribute errors are raise once the graph is populated using from_cudf_edgelist

Reply:
This might be a better example of my concern:
```
>>> G.from_cudf_edglist(df, source='0', destination='1', edge_attr='2', renumber=False)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/opt/conda/envs/rapids/lib/python3.8/site-packages/cugraph-0.19.0a0+106.g839f98fb-py3.8-linux-x86_64.egg/cugraph/structure/graph_classes.py", line 57, in __getattr__
    raise Exception("Graph is Empty")
Exception: Graph is Empty
```
The user had a typo (`from_cudf_edglist`), but I think there's a good chance we'd get a bug report.

Reply:
updated to always raise Attribute Error